### PR TITLE
Community Forum Code of Conduct update

### PR DESCRIPTION
**Pull request by vollmera**
updates to Community Forum Code of Conduct, including
- syncing with sections of Community Guidelines that have gotten out of sync over time
- noting that it's not ok to create additional free accounts to ask about flagged or suspended accounts in the community forum
- noting that you should use the forum or the Support contact form, and not contact any member of GitHub Staff via unsolicited mentions or pings, or other channels

### CODE REVIEW

File: Policies/github-community-forum-code-of-conduct.md [hunk 1]
```diff
@@ -63,28 +63,36 @@ The following are not hard and fast rules, merely aids to the human judgment of
...
 
 - **Other conduct which could reasonably be considered inappropriate in a professional setting.** The GitHub Community Forum is a professional space and should be treated as such.
 
+- **Violation of Terms of Service.** If your Github.com account is identified in violation of [Terms of Service](/github/site-policy/github-terms-of-service/) we will have to close your GitHub Community Forum as well.  [Terms of Service](/github/site-policy/github-terms-of-service/) also indicates that "One person or legal entity may maintain no more than one free Account" therefore additional free accounts created to inquire about flagged or suspended accounts in the community forum will also be removed.
```
Comment (lee-dohm) [inline Policies/github-community-forum-code-of-conduct.md:94]:
```suggestion
- **Violation of Terms of Service.** If your Github.com account is identified in violation of [Terms of Service](/github/site-policy/github-terms-of-service/) we will have to close your GitHub Community Forum account as well.  [Terms of Service](/github/site-policy/github-terms-of-service/) also indicates that "One person or legal entity may maintain no more than one free Account" therefore additional free accounts created to inquire about flagged or suspended accounts in the community forum will also be removed.
```

**Review by lee-dohm — APPROVED** [review]
Other than the one suggestion I gave, this looks good to me 👍

**Comment by Vut11** [comment]
SARAWUT

File: Policies/github-community-forum-code-of-conduct.md [hunk 2]
```diff
@@ -63,28 +63,36 @@ The following are not hard and fast rules, merely aids to the human judgment of
...
+- **Hate speech and discrimination.** While it is not forbidden to broach topics such as age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation, we do not tolerate speech that attacks a person or group of people on the basis of who they are. Just realize that when approached in an aggressive or insulting manner, these (and other) sensitive topics can make others feel unwelcome, or perhaps even unsafe. While there's always the potential for misunderstandings, we expect our community members to remain respectful and civil when discussing sensitive topics.
```
Comment (KOLANICH) [inline Policies/github-community-forum-code-of-conduct.md:68]: `politics` is missing from the list.

**Pull request merged**